docs/content/develop/step-2c.md

```diff
@@ -25,37 +30,123 @@ Your Sass file will initially import additional Sass from the pfe-sass node modu
 }
 ```
 
+When structuring your Sass, a good rule of thumb for organization is:
+
+```
+// Define your hooks at the top
+$LOCAL-VARIABLES: (
+  Property: default,
+  region: (
+    Property: default
+  )
+);
+
+:host {
+  // Apply the hooks to the appropriate properties
+  property: pfe-local(Property);
+}
+
+// Modifiers should update local variables
+:host([modifiers]) {
+  --pfe-cool-element--Property: updated;
```

Comment:
what does `updated` mean here?

Reply:
ohh.. 💡 
what if the examples said:
```
    Property: default-value

  --pfe-cool-element--Property: updated-value;
```

Reply:
+1, added them as suggestions so I can commit them from the UI in one batch